Description: Handle utf8-data, CreationDate and ModDate fields in update_info correctly
Debian-Bug: http://bugs.debian.org/424920
Debian-Bug: http://bugs.debian.org/461169
Author: Quentin Godfroy <godfroy@clipper.ens.fr>, Chris Adams <cadams@salk.edu>

--- a/pdftk/report.cc
+++ b/pdftk/report.cc
@@ -1233,7 +1233,7 @@
 								string_to_jcharstring( jvs, jvs_size, &jvs_len, it->second );
 
 								info_p->put( new itext::PdfName( JvNewStringUTF(it->first.c_str()) ),
-														 new itext::PdfString( JvNewString(jvs, jvs_len), itext::PdfObject::TEXT_UNICODE ) );
+														 new itext::PdfString( JvNewStringUTF((char* )it->second.c_str()), (strcmp(it->first.c_str(), "ModDate") && strcmp(it->first.c_str(), "CreationDate")) ? itext::PdfObject::TEXT_UNICODE : itext::PdfObject::TEXT_PDFDOCENCODING ) );
 							}
 						}
 				}
